conn_linux.go

```diff
@@ -226,6 +226,16 @@ func (c *conn) SetBPF(filter []bpf.RawInstruction) error {
 	)
 }
 
+// DetachBPF removes a BPF filter from a conn.
+func (c *conn) DetachBPF() error {
+	var dummy int
+	return c.s.SetSockopt(
+		unix.SOL_SOCKET,
+		unix.SO_DETACH_FILTER,
+		unsafe.Pointer(&dummy),
+		uint32(unsafe.Sizeof(dummy)))
```

Comment:
Add a newline before the final brace so it lines up nicely at the first level of indentation.